runtime/integration-tests/Cargo.toml

```diff
@@ -22,6 +22,7 @@ serde = { workspace = true }
 thiserror = { workspace = true }
 tokio = { workspace = true }
 tracing-subscriber = { workspace = true }
+log = { workspace = true }
```

Comment:
Nit: Do we need that?

Reply:
Not, but it's true that it's confortable to have it there not to add that crate each time you need to add logs to debug something.


BTW, I think we will close this PR and will go with rococo